test/browser/question.test.js

```diff
@@ -0,0 +1,33 @@
+const { expect } = require('test/chai-sinon');
+const { startServices } = require('test/browser/common');
+const mockData = require('test/mock/question').template;
+const QuestionPage = require('test/page-objects/question');
+
+describe('Question page', () => {
+  /* eslint-disable init-declarations */
+  let page;
+  let questionPage;
+  /* eslint-enable init-decalarations */
+
+  before(async() => {
+    const res = await startServices();
+    page = res.page;
+    questionPage = new QuestionPage(page);
+    await questionPage.visitPage();
+  });
+
+  after(async() => {
+    if (page && page.close) {
+      await page.close();
+    }
+  });
+
+  it('is on the /question path', () => {
+    questionPage.verifyPage();
+  });
+
+  it('has question heading from api request', async() => {
+    await questionPage.screenshot('question');
+    expect(await questionPage.getHeading()).to.equal(mockData.question_header_text);
```

Comment:
We need to think about how this will work when running `yarn test:functional` against environments such as preview or AAT. Probably need to have a discussion with the Chris's to work out how this will work.

Reply:
Yeah true. We might have to do something similar as TYA where there is test data in the other environments which our tests run against and we make sure dyson is only used locally. But it means the dyson mock data might have to match what is in the test data on the other envs

Reply:
Yeah that's pretty much what I was thinking.